plugins/Marketplace/resources/qml/LicenseDialog.qml

```diff
@@ -0,0 +1,92 @@
+//Copyright (c) 2021 Ultimaker B.V.
+//Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Dialogs 1.1
+import QtQuick.Window 2.2
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+import QtQuick.Controls.Styles 1.4
```

Comment:
Remove this. We shouldn't be using controls styles 1 (because we will need to remove it later!)

Reply:
Fixed in f993243d57bf030f78f8e1e8383428e990349db0